### ISSUE TITLE
example-inference-1 not running. Pre trained Model error

### ISSUE THREAD

**Issue by PrinceP** [tensorlayer/openpose-plus#136]
While running 
python3 examples/example-inference-1.py --path-to-npz models/pretrained-models/models/openpose-plus/hao28-pose600000.npz  --images data/mscoco2017/val2017/ --limit 1

Following error is encountered
File "/usr/local/lib/python3.5/dist-packages/numpy/lib/npyio.py", line 440, in load
    return pickle.load(fid, **pickle_kwargs)
_pickle.UnpicklingError: invalid load key, 'v'.

**Comment by mandeman** [comment]
Is it because you are trying to pass the folder in --images instead of individual comma separated filenames?

**Comment by PrinceP** [comment]
@mandeman 
I modified the script to run for the images in a folder. 
I trained a new vggtiny model again for some iterations, used the script command

python3 examples/example-inference-1.py --path-to-npz models/pose30000.npz   --base-model=vggtiny  --images data/mscoco2017/val2017/ --limit 1
And got the output
--------------------------------------------------------------------------------
tot (s)      count        mean (ms)    name
--------------------------------------------------------------------------------
3.933178     1            3933.177710  main
1.996399     1            1996.399403  create TfPoseEstimator
1.891422     1            1891.421556  e.inference
0.007528     1            7.527828     read_imgfile

But for uploaded hao28-pose600000.npz, its giving error.

**Comment by lgarithm** [comment]
what is the sha1sum of `hao28-pose600000.npz` on your computer?

mine is 
```
0962749be446b10fd2843efcaab1cc5f3fd92273  hao28-pose600000.npz
```

**Comment by PrinceP** [comment]
b8bd3b50edb97d9c6abfbda7a0a56f75a3b7c334  hao28-pose600000.npz

Can you give the weights.

**Comment by lgarithm** [comment]
which means `cat hao28-pose600000.npz` yields
```
version https://git-lfs.github.com/spec/v1
oid sha256:3fdfd66b709653b62a7ac2906f4a4d94898e4bf909e376b91cde20e99041f905
size 34584656
```

Because that file is managed by [git lfs](https://git-lfs.github.com)

You can download the content of that file using the URL obtained from web:

https://github.com/tensorlayer/pretrained-models/raw/master/models/openpose-plus/hao28-pose600000.npz

**Comment by PrinceP** [comment]
Worked fine. 
Had some questions about the inference benchmarks. Will open another issue. 
Thanks

**Comment by karthee320** [comment]
I have downloaded the model referenced above and run the command 

python3 examples/example-inference-1.py --path-to-npz=models/hao28-pose600000.npz --images=data/mscoco2017/val2017/000000000785.jpg --plot=True

But it is giving errors.

tensorflow.python.framework.errors_impl.InvalidArgumentError: Dimension 0 in both shapes must be equal, but are 7 and 3. Shapes are [7,7,128,128] and [3,3,128,128]. for 'Assign' (op: 'Assign') with input shapes: [7,7,128,128], [3,3,128,128].

 I tried all the three model name(vgg, vggtiny,mobilenet) as Command Line argument for base-model. One of the dimensions were not equal. Can anyone know what is the reason?
